lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
I was going to suggest moving `Looper.loop()` into the `if` block at the top of this method along with `Looper.prepare()` however when I tested that change it the location updates were never delivered to the service. Any idea why this needs to be after the location requests are created?